x/tier/keeper/portfolio.go

```diff
@@ -222,6 +222,18 @@ func (k Keeper) RetreiveLeverageLpTotal(ctx sdk.Context, user sdk.AccAddress) sd
 	return totalValue
 }
 
+func (k Keeper) RetreiveConsolidatedPrice(ctx sdk.Context, denom string) (sdk.Dec, error) {
```

Comment:
@amityadav0 small typo with the word `retrieve`

Reply:
thanks, fixed